ghost/core/test/e2e-browser/admin/publishing.spec.js

```diff
@@ -58,7 +66,7 @@ const publishPost = async (page, {type = 'publish', time} = {}) => {
 
     // set the publish type
     await page.locator('[data-test-setting="publish-type"] > button').click();
-    await page.locator(`[data-test-publish-type="${type}"]`).setChecked(true);
+    await page.locator(`[data-test-publish-type="${type}"]`).setChecked(true, {timeout: 50});
```

Comment:
@kevinansfield  The `locator(`[data-test-publish-type="${type}"]`)` doesn't really work once Mailgun is enabled. With all three publishing options enabled the selector fails to choose the right option. When Mailgun is disabled, the selector chose the only available button and moved on.

Reply:
I've pushed a hacky fix with "findByText" selectors for this, so I could debug Mailgun further